source/OutfitterPanel.cpp

```diff
@@ -766,50 +773,61 @@ void OutfitterPanel::DrawKey()
 	const Font &font = FontSet::Get(14);
 	Color color[2] = {*GameData::Colors().Get("medium"), *GameData::Colors().Get("bright")};
 	const Sprite *box[2] = {SpriteSet::Get("ui/unchecked"), SpriteSet::Get("ui/checked")};
-	
-	Point pos = Screen::BottomLeft() + Point(10., -30.);
+
+	Point pos = Screen::BottomLeft() + Point(10., -VISIBILITY_CHECKBOXES_SIZE + 10.);
```

Comment:
You made this variable protected, and it also has an associated virtual getter function. Use the getter instead of the variable (probably the variable is not even needed).